### PR TITLE
Tag `example-bad` blocks with html-nolint

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- Updating example-bad code blocks with the new tag. Refer: https://github.com/mdn/yari/pull/7017

### CODE REVIEW

File: files/en-us/learn/accessibility/html/index.md [hunk 1]
```diff
@@ -398,18 +399,15 @@ Make sure your labels make sense out of context, read on their own, as well as i
 
 but this is bad link text:
 
-```html example-bad
-<p>
-  Whales are really awesome creatures. To find out more about whales,
-  <a href="whales.html">click here</a>.
-</p>
+```html-nolint example-bad
+<p>Whales are really awesome creatures. To find out more about whales, <a href="whales.html">click here</a>.</p>
```
Comment (OnkarRuikar) [inline files/en-us/learn/accessibility/html/index.md:403]:
```suggestion
```html-nolint example-bad
<p>
  Whales are really awesome creatures. To find out more about whales,
    <a href="whales.html">click here</a>.</p>
```

File: files/en-us/learn/accessibility/html/index.md [hunk 2]
```diff
@@ -398,18 +399,17 @@ Make sure your labels make sense out of context, read on their own, as well as i
 
 but this is bad link text:
 
-```html example-bad
+```html-nolint example-bad
 <p>
   Whales are really awesome creatures. To find out more about whales,
-  <a href="whales.html">click here</a>.
-</p>
+    <a href="whales.html">click here</a>.</p>
```
Comment (OnkarRuikar) [inline files/en-us/learn/accessibility/html/index.md:405]:
```suggestion
  <a href="whales.html">click here</a>.
</p>
```

**Review by estelle — COMMENTED** [review]
Looks good, but there are conflicts.

File: files/en-us/learn/accessibility/html/index.md [hunk 3]
```diff
@@ -399,7 +400,7 @@ Make sure your labels make sense out of context, read on their own, as well as i
 
 but this is bad link text:
 
-```html example-bad
+```html-nolint example-bad
```
Comment (teoli2003) [inline files/en-us/learn/accessibility/html/index.md:403]: Same here: we don't need -nolint.

File: files/en-us/learn/accessibility/html/index.md [hunk 4]
```diff
@@ -410,7 +411,7 @@ but this is bad link text:
 
 Form labels are also important for giving you a clue about what you need to enter into each form input. The following seems like a reasonable enough example:
 
-```html example-bad
+```html-nolint example-bad
 Fill in your name: <input type="text" id="name" name="name" />
```
Comment (teoli2003) [inline files/en-us/learn/accessibility/html/index.md:415]: No nolint here

File: files/en-us/learn/getting_started_with_the_web/html_basics/index.md [hunk 5]
```diff
@@ -66,7 +66,7 @@ You can put elements inside other elements too — this is called **nesting**. I
 
 You do however need to make sure that your elements are properly nested. In the example above, we opened the {{htmlelement("p")}} element first, then the {{htmlelement("strong")}} element; therefore, we have to close the {{htmlelement("strong")}} element first, then the {{htmlelement("p")}} element. The following is incorrect:
 
-```html example-bad
+```html-nolint example-bad
```
Comment (teoli2003) [inline files/en-us/learn/getting_started_with_the_web/html_basics/index.md:69]: The nolint here is needed.

File: files/en-us/learn/html/introduction_to_html/creating_hyperlinks/index.md [hunk 6]
```diff
@@ -226,7 +226,7 @@ Let's look at a specific example:
 
 **Bad** link text: [Click here](https://www.mozilla.org/firefox/) to download Firefox
 
-```html example-bad
+```html-nolint example-bad
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/creating_hyperlinks/index.md:229]: THis no-lint is not needed

File: files/en-us/learn/javascript/first_steps/variables/index.md [hunk 7]
```diff
@@ -72,7 +72,7 @@ In this example pressing the button runs some code. The first line pops a box up
 
 To understand why this is so useful, let's think about how we'd write this example without using a variable. It would end up looking something like this:
 
-```html example-bad
+```html-nolint example-bad
```
Comment (teoli2003) [inline files/en-us/learn/javascript/first_steps/variables/index.md:75]: I think nolint is not needed hre

File: files/en-us/learn/javascript/first_steps/what_is_javascript/index.md [hunk 8]
```diff
@@ -310,7 +310,7 @@ function createParagraph() {
 }
 ```
 
-```html example-bad
+```html-nolint example-bad
```
Comment (teoli2003) [inline files/en-us/learn/javascript/first_steps/what_is_javascript/index.md:313]: Not needed

File: files/en-us/learn/tools_and_testing/cross_browser_testing/accessibility/index.md [hunk 9]
```diff
@@ -80,8 +80,9 @@ Semantic HTML (where the elements are used for their correct purpose) is accessi
 
 The most important quick win in semantic HTML is to use a structure of headings and paragraphs for your content; this is because screen reader users tend to use the headings of a document as signposts to find the content they need more quickly. If your content has no headings, all they will get is a huge wall of text with no signposts to find anything. Examples of bad and good HTML:
 
-```html example-bad
-<font size="7">My heading</font> <br /><br />
+```html-nolint example-bad
```
Comment (teoli2003) [inline files/en-us/learn/tools_and_testing/cross_browser_testing/accessibility/index.md:83]: Not needed.

File: files/en-us/mdn/writing_guidelines/writing_style_guide/code_style_guide/html/index.md [hunk 10]
```diff
@@ -113,7 +113,7 @@ Use semantic class/ID names, and separate multiple words with hyphens. Don't use
 <p class="editorial-summary">Blah blah blah</p>
 ```
 
-```html example-bad
+```html-nolint example-bad
```
Comment (teoli2003) [inline files/en-us/mdn/writing_guidelines/writing_style_guide/code_style_guide/html/index.md:116]: Not needed

File: files/en-us/mdn/writing_guidelines/writing_style_guide/code_style_guide/html/index.md [hunk 11]
```diff
@@ -141,7 +141,7 @@ As an example, you could just write:
 
 Instead of:
 
-```html example-bad
+```html-nolint example-bad
```
Comment (teoli2003) [inline files/en-us/mdn/writing_guidelines/writing_style_guide/code_style_guide/html/index.md:144]: Not needed

File: files/en-us/web/html/element/a/index.md [hunk 12]
```diff
@@ -369,16 +369,20 @@ Using `target="_blank"` without [`rel="noreferrer"`](/en-US/docs/Web/HTML/Link_t
 
 A sadly common mistake is to only link the words "click here" or "here":
 
-```html example-bad
-<p>Learn more about our products <a href="/products">here</a>.</p>
+```html-nolint example-bad
```
Comment (teoli2003) [inline files/en-us/web/html/element/a/index.md:372]: Not needed

File: files/en-us/mdn/writing_guidelines/writing_style_guide/code_style_guide/index.md [hunk 13]
```diff
@@ -192,7 +192,7 @@ To represent the code block as a good or bad example, add `example-good` or `exa
 <p class="brush: js example-good">
 ```
 
-```html example-bad
+```html-nolint example-bad
```
Comment (teoli2003) [inline files/en-us/mdn/writing_guidelines/writing_style_guide/code_style_guide/index.md:195]: No nolint here. (We should likely elaborate when we want it there, but in general case, we don't want it)

File: files/en-us/mdn/writing_guidelines/writing_style_guide/code_style_guide/index.md [hunk 14]
```diff
@@ -203,6 +203,6 @@ These will be rendered as:
...
```
Comment (teoli2003) [inline files/en-us/mdn/writing_guidelines/writing_style_guide/code_style_guide/index.md:206]: No nolint

File: files/en-us/web/html/element/heading_elements/index.md [hunk 15]
```diff
@@ -150,7 +150,7 @@ A common navigation technique for users of screen reading software is jumping fr
 
 #### Don't
 
-```html example-bad
+```html-nolint example-bad
```
Comment (teoli2003) [inline files/en-us/web/html/element/heading_elements/index.md:153]: No nolint

File: files/en-us/web/html/element/img/index.md [hunk 16]
```diff
@@ -302,7 +302,7 @@ An `alt` attribute's value should clearly and concisely describe the image's con
 
 #### Don't
 
-```html example-bad
+```html-nolint example-bad
```
Comment (teoli2003) [inline files/en-us/web/html/element/img/index.md:305]: No nolint

File: files/en-us/web/html/element/label/index.md [hunk 17]
```diff
@@ -90,7 +90,7 @@ Don't place interactive elements such as {{HTMLElement("a", "anchors")}} or {{HT
 
 #### Don't
 
-```html example-bad
+```html-nolint example-bad
```
Comment (teoli2003) [inline files/en-us/web/html/element/label/index.md:93]: I think nolint is not necessary there

File: files/en-us/web/html/element/label/index.md [hunk 18]
```diff
@@ -117,7 +117,7 @@ If a [form](/en-US/docs/Web/HTML/Element/form), or a section of a form needs a t
...
```
Comment (teoli2003) [inline files/en-us/web/html/element/label/index.md:120]: No nolint there

File: files/en-us/web/http/headers/content-language/index.md [hunk 19]
```diff
@@ -72,7 +72,7 @@ The global [`lang`](/en-US/docs/Web/HTML/Global_attributes/lang) attribute is us
 
 Do **not** use this meta element like this for stating a document language:
 
-```html example-bad
+```html-nolint example-bad
```
Comment (teoli2003) [inline files/en-us/web/http/headers/content-language/index.md:75]: No nolint

File: files/en-us/web/http/headers/content-security-policy/base-uri/index.md [hunk 20]
```diff
@@ -72,7 +72,7 @@ add_header Content-Security-Policy "base-uri 'self';"
 
 Since your domain isn't `example.com`, a {{HTMLElement("base")}} element with its `href` set to `https://example.com` will result in a CSP violation.
 
-```html example-bad
+```html-nolint example-bad
```
Comment (teoli2003) [inline files/en-us/web/http/headers/content-security-policy/base-uri/index.md:75]: No nolint

File: files/en-us/web/http/headers/content-security-policy/form-action/index.md [hunk 21]
```diff
@@ -77,8 +77,8 @@ add_header Content-Security-Policy "form-action 'none';"
 
 Using a {{HTMLElement("form")}} element with an action set to inline JavaScript will result in a CSP violation.
 
-```html example-bad
-<meta http-equiv="Content-Security-Policy" content="form-action 'none'" />
+```html-nolint example-bad
```
Comment (teoli2003) [inline files/en-us/web/http/headers/content-security-policy/form-action/index.md:80]: No nolint

File: files/en-us/web/http/headers/content-security-policy/navigate-to/index.md [hunk 22]
```diff
@@ -72,7 +72,7 @@ Note that this same set of values can be used in all {{Glossary("fetch directive
 Using a {{HTMLElement("form")}} element with an action set to inline JavaScript will
 result in a CSP violation.
 
-```html example-bad
+```html-nolint example-bad
```
Comment (teoli2003) [inline files/en-us/web/http/headers/content-security-policy/navigate-to/index.md:75]: No nolint

File: files/en-us/web/http/headers/content-security-policy/require-sri-for/index.md [hunk 23]
```diff
@@ -55,7 +55,7 @@ valid integrity attribute.
 
 However, scripts without integrity won't load anymore:
 
-```html example-bad
+```html-nolint example-bad
```
Comment (teoli2003) [inline files/en-us/web/http/headers/content-security-policy/require-sri-for/index.md:58]: No nolint

File: files/en-us/web/privacy/firefox_tracking_protection/index.md [hunk 24]
```diff
@@ -53,7 +53,7 @@ More subtly, if other parts of your site depend on trackers being loaded, then t
 
 For example, you should not use Google Analytics in the following way:
 
-```html example-bad
+```html-nolint example-bad
```
Comment (teoli2003) [inline files/en-us/web/privacy/firefox_tracking_protection/index.md:56]: No nolint

**Review by teoli2003 — COMMENTED** [review]
Quite a few nolint here are not needed: let's be defensive, only add it when Prettier and/or Markdown will make the bad example moot.

File: files/en-us/learn/accessibility/html/index.md (hunk 3 above)
Comment (OnkarRuikar) [inline files/en-us/learn/accessibility/html/index.md:403]:
```suggestion
```html example-bad
```

File: files/en-us/learn/accessibility/html/index.md [hunk 25]
```diff
@@ -410,7 +411,7 @@ but this is bad link text:
 
 Form labels are also important for giving you a clue about what you need to enter into each form input. The following seems like a reasonable enough example:
 
-```html example-bad
+```html-nolint example-bad
```
Comment (OnkarRuikar) [inline files/en-us/learn/accessibility/html/index.md:414]:
```suggestion
```html example-bad
```

File: files/en-us/learn/html/introduction_to_html/creating_hyperlinks/index.md (hunk 6 above)
Comment (OnkarRuikar) [inline files/en-us/learn/html/introduction_to_html/creating_hyperlinks/index.md:229]:
```suggestion
```html example-bad
```

File: files/en-us/learn/javascript/first_steps/variables/index.md (hunk 7 above)
Comment (OnkarRuikar) [inline files/en-us/learn/javascript/first_steps/variables/index.md:75]:
```suggestion
```html example-bad
```

File: files/en-us/learn/javascript/first_steps/what_is_javascript/index.md (hunk 8 above)
Comment (OnkarRuikar) [inline files/en-us/learn/javascript/first_steps/what_is_javascript/index.md:313]:
```suggestion
```html example-bad
```

File: files/en-us/learn/tools_and_testing/cross_browser_testing/accessibility/index.md (hunk 9 above)
Comment (OnkarRuikar) [inline files/en-us/learn/tools_and_testing/cross_browser_testing/accessibility/index.md:83]:
```suggestion
```html example-bad
```

File: files/en-us/mdn/writing_guidelines/writing_style_guide/code_style_guide/html/index.md (hunk 10 above)
Comment (OnkarRuikar) [inline files/en-us/mdn/writing_guidelines/writing_style_guide/code_style_guide/html/index.md:116]:
```suggestion
```html example-bad
```

File: files/en-us/mdn/writing_guidelines/writing_style_guide/code_style_guide/html/index.md (hunk 11 above)
Comment (OnkarRuikar) [inline files/en-us/mdn/writing_guidelines/writing_style_guide/code_style_guide/html/index.md:144]:
```suggestion
```html example-bad
```

File: files/en-us/mdn/writing_guidelines/writing_style_guide/code_style_guide/index.md (hunk 13 above)
Comment (OnkarRuikar) [inline files/en-us/mdn/writing_guidelines/writing_style_guide/code_style_guide/index.md:195]:
```suggestion
```html example-bad
```

File: files/en-us/mdn/writing_guidelines/writing_style_guide/code_style_guide/index.md (hunk 14 above)
Comment (OnkarRuikar) [inline files/en-us/mdn/writing_guidelines/writing_style_guide/code_style_guide/index.md:206]:
```suggestion
```html example-bad
```

File: files/en-us/web/html/element/a/index.md (hunk 12 above)
Comment (OnkarRuikar) [inline files/en-us/web/html/element/a/index.md:372]:
```suggestion
```html example-bad
```

File: files/en-us/web/html/element/heading_elements/index.md (hunk 15 above)
Comment (OnkarRuikar) [inline files/en-us/web/html/element/heading_elements/index.md:153]:
```suggestion
```html example-bad
```

File: files/en-us/web/html/element/img/index.md (hunk 16 above)
Comment (OnkarRuikar) [inline files/en-us/web/html/element/img/index.md:305]:
```suggestion
```html example-bad
```

File: files/en-us/web/html/element/label/index.md (hunk 17 above)
Comment (OnkarRuikar) [inline files/en-us/web/html/element/label/index.md:93]:
```suggestion
```html example-bad
```

File: files/en-us/web/html/element/label/index.md (hunk 18 above)
Comment (OnkarRuikar) [inline files/en-us/web/html/element/label/index.md:120]:
```suggestion
```html example-bad
```

File: files/en-us/web/http/headers/content-language/index.md (hunk 19 above)
Comment (OnkarRuikar) [inline files/en-us/web/http/headers/content-language/index.md:75]:
```suggestion
```html example-bad
```

File: files/en-us/web/http/headers/content-security-policy/base-uri/index.md (hunk 20 above)
Comment (OnkarRuikar) [inline files/en-us/web/http/headers/content-security-policy/base-uri/index.md:75]:
```suggestion
```html example-bad
```

File: files/en-us/web/http/headers/content-security-policy/form-action/index.md (hunk 21 above)
Comment (OnkarRuikar) [inline files/en-us/web/http/headers/content-security-policy/form-action/index.md:80]:
```suggestion
```html example-bad
```

File: files/en-us/web/http/headers/content-security-policy/navigate-to/index.md (hunk 22 above)
Comment (OnkarRuikar) [inline files/en-us/web/http/headers/content-security-policy/navigate-to/index.md:75]:
```suggestion
```html example-bad
```

File: files/en-us/web/http/headers/content-security-policy/require-sri-for/index.md (hunk 23 above)
Comment (OnkarRuikar) [inline files/en-us/web/http/headers/content-security-policy/require-sri-for/index.md:58]:
```suggestion
```html example-bad
```

File: files/en-us/web/privacy/firefox_tracking_protection/index.md (hunk 24 above)
Comment (OnkarRuikar) [inline files/en-us/web/privacy/firefox_tracking_protection/index.md:56]:
```suggestion
```html example-bad
```

File: files/en-us/learn/accessibility/html/index.md [hunk 26]
```diff
@@ -369,8 +370,7 @@ While the above addition allows us to tab to the buttons, it does not allow us t
 
 ```js
 document.onkeydown = (e) => {
-  // The Enter/Return key
-  if (e.key === "Enter") {
+  if (e.key === "Enter") { // The Enter/Return key
```
Comment (OnkarRuikar) [inline files/en-us/learn/accessibility/html/index.md:373]:
```suggestion
  // The Enter/Return key
  if (e.key === "Enter") {
```

**Comment by OnkarRuikar** [comment]
Thanks @estelle @teoli2003 !

This was supposed to land before the Prettier PRs, that is why so many conflicts. I've rebased it now.

After removing the unwanted `-nolint` tags I re-ran the Prettier. Now some of those freed up code blocks are prettier.

After this PR lands, in the entire repo, there will be only:
- 22 ```` ```html example-bad```` blocks, which are eligible for prettier.
- 10 ```` ```html-nolint example-bad```` blocks.

In total there are only 33 bad blocks for HTML, and all got sorted out here.

File: files/en-us/learn/accessibility/html/index.md [hunk 27]
```diff
@@ -126,8 +126,9 @@ We've prepared a version with longer text for you to try out with a screen reade
 
 People sometimes write headings, paragraphs, etc. using line breaks and adding HTML elements purely for styling, something like the following:
 
-```html example-bad
-<span style="font-size: 3em">My heading</span> <br /><br />
+```html-nolint example-bad
```
Comment (teoli2003) [inline files/en-us/learn/accessibility/html/index.md:129]:
```suggestion
```html example-bad
```

**Review by teoli2003 — APPROVED** [review]

**Comment by adFuckYou** [comment]
Sorry guys, he’s dooing it to help me, and I don’t even know him, there’s like 3 hackers in my wallet and not like there’s valuebles in it, but it’s all I’ve worked for this far, and no I didn’t click on any links or something stupid, I did that when I started and got wiped in 10 seconds. This is my love, my life. And I had no clue what to do about it

**Comment by adFuckYou** [comment]
It’s my first time on here and seeing how mam of you contribute and help, and that’s just amazing! I appreciate it, but I don’t appreciate hackers, one with a Wallet balance of $10 billion trying to come and steal my $100. Why? Why? Use that knowledge and use if for the better

**Comment by adFuckYou** [comment]
And I only rleaiised when I download Firefox since I removed the extensions from chrome or previously, and inspection picked up a the errors

**Comment by adFuckYou** [comment]
So if anyone’s knowledgeable enough to compile a report for magic Eden, half or even 60% of the bounty is yours, that’s $500 k in total, they said they can’t do anything about the 40-50 NFTs I was holding and just vanished before I could even try and revoke… they just sorry and move your listed ones to a new wallet, which would cost money to transfer, wouldn’t even offer me a small amount to create an new wallet to tranfer since I was flat out broke

**Pull request merged**